Tests/test_PDB.py

```diff
@@ -1231,14 +1257,14 @@ def test_entitity_copy(self):
 
 
 def eprint(*args, **kwargs):
-    '''Helper function that prints to stderr.'''
+    """Helper function that prints to stderr."""
     print(*args, file=sys.stderr, **kwargs)
 
 
 def will_it_float(s):
-    '''
+    """
```

Comment:
Maybe we should file an enhancement request with ``pydocstyle`` to check this...